pages/generate-test-cases.tsx

```diff
@@ -36,7 +36,7 @@ const TestCaseEditorPage: NextPage = () => {
               mb,
               start?.toISOString(),
               end?.toISOString(),
-              useTrustMetaProfile
+              trustMetaProfile
```

Comment:
Same issue as other situations that use this function.

Reply:
Never mind! Github was showing me an out of date copy of the `calculateDetailedResult` function.